examples/chem-sync-local-flask/README.md

```diff
@@ -81,6 +81,7 @@ https://brave-wombats-poke.loca.lt
 2. Ensure you've been granted access to the [Benchling Developer Platform Capability](https://help.benchling.com/hc/en-us/articles/9714802977805-Access-the-Benchling-Developer-Platform).
 3. This example also requires a [Lab Automation](https://www.benchling.com/resources/benchling-lab-automation) license.
 4. [Molecule entities](https://help.benchling.com/hc/en-us/articles/9684254682893-Molecule-entity-overview) will need to be enabled on your tenant.
+5. [Global Apps](https://docs.benchling.com/docs/global-apps-faq) will need to be enabled on your tenant.
```

Comment:
@Psterne and @dgervais-benchling for doc update review of `README.md`.

Also, some of the screenshot GIFs in here reference "older" App screens. It's probably fine for now, but maybe something we should redo once Global Apps is live everywhere.

Reply:
The updates look good to me. We will almost certainly forget to update the link if we ever take down the FAQ 😅 

And yes, at some point we should make sure the screenshots here all show Global Apps. I think it's fine for now though since we are still iterating on the Global Apps UI anyway.